spyder/utils/programs.py

```diff
@@ -358,8 +358,12 @@ def _get_win_applications():
         if software[None]:
             key = software['key'].capitalize().replace('.exe', '')
             expanded_fpath = os.path.expandvars(software[None]).lower()

```

Comment:
@CAM-Gerlach could you add a print here on your code and paste the output?

`print([expanded_fpath])`